lib/components/src/blocks/Source.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Source, SourceError } from './Source';
+import { DocsPageWrapper } from './DocsPage';
+
+export default {
+  title: 'Docs|Source',
+  Component: Source,
+  decorators: [getStory => <DocsPageWrapper>{getStory()}</DocsPageWrapper>],
+};
+
+export const noStory = () => <Source error={SourceError.NO_STORY} />;
+noStory.title = 'no story';
+
+export const sourceUnavailable = () => <Source error={SourceError.SOURCE_UNAVAILABLE} />;
+sourceUnavailable.title = 'source unavailable';
+
+const jsxCode = `
+<MyComponent boolProp scalarProp={1} complexProp={{ foo: 1, bar: '2' }}>
+  <SomeOtherComponent funcProp={(a) => a.id} />
+</MyComponent>
+`.trim();
```

Comment:
these trims are causing the bad indenting, because the formatter expects the whitespace from template literals.